File: 240722/대문자로 바꾸기/change-to-capital.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        // Scanner
        Scanner sc = new Scanner(System.in);
        // 배열 선언
        int[][] A = new int[5][3];
        // input
        for (int i =0; i <5; i++){
            for (int j = 0; j<3; j++){
                String str = sc.next();
                System.out.print(str.toUpperCase() + " ");
            }
            System.out.println();
        }
    }
}

File: 250317/대문자로 바꾸기/change-to-capital.kt
import java.io.*;

fun main(){
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.`out`))

    (0 .. 4).forEach{ index ->
        val input = br.readLine().split(" ").map{ it.toUpperCase() }
        bw.write(input.joinToString(" ") + "\n")
    }

    bw.flush()
    bw.close()
}